mk/uvmt/dsim.mk

```diff
@@ -345,6 +345,13 @@ corev-dv: clean_riscv-dv \
 	  clone_riscv-dv \
 	  comp_corev-dv
 
+
+################################################################################
```

Comment:
I see that there is a similar code segment in each of the simulator-specified Makefiles.  It would ease maintenance considerably if this code could be placed in Common.mk.